node/src/accountant/db_big_integer/mod.rs

```diff
@@ -2,5 +2,5 @@
 
 pub mod big_int_db_processor;
 pub mod big_int_divider;
-#[cfg(test)]
-mod test_utils;
+
+pub mod test_utils;
```

Comment:
I don't know why it was made public. Maybe it could be prevented. I think it would be good if we manage to do that.

Reply:
Because there is a mock that I use under a different module. It's okay, I think